### PR TITLE
Prepare release 4 .5.0

### PR DESCRIPTION
**Pull request by vegaro**
- This release has some changes that should help prevent "There is no singleton instance" errors.
    - Added `isConfigured` function to be able to check if the instance of Purchases has been configured before calling any function that accesses the singleton instance.
    - Some functions that were returning `void`, now return `Promise<void>`: `setAllowSharingStoreAccount`, `setFinishTransactions`, `setSimulatesAskToBuyInSandbox`, `addAttributionData`, `setDebugLogsEnabled`, `syncPurchases`, `setAutomaticAppleSearchAdsAttributionCollection`, `invalidatePurchaserInfoCache`, `presentCodeRedemptionSheet`, and all the subscriber attributes related functions.
    - Accessing a function that accesses the singleton instance before it has been setup will now reject with a `UninitializedPurchasesError`.
    - Related issue: https://github.com/RevenueCat/react-native-purchases/issues/101
    - Related PR: https://github.com/RevenueCat/react-native-purchases/pull/310

- Bump`purchases-hybrid-common` to `1.11.1` [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/1.10.1)
- Bump `purchases-ios` to `3.13.1` ([Changelog here](https://github.com/RevenueCat/purchases-ios/releases/3.13.1))
- Bump `purchases-android` to `4.6.0` ([Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.6.0))

### CODE REVIEW

**Review by taquitos — APPROVED** [review]
🪢. 🐐

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
-- Add canMakePayments method (https://github.com/RevenueCat/react-native-purchases/pull/244)
-- Add PR template (https://github.com/RevenueCat/react-native-purchases/pull/263)
+- This release has some changes that should help prevent "There is no singleton instance" errors.
+    - Added `isConfigured` function to be able to check if the instance of Purchases has been configured before calling any function that accesses the singleton instance.
+    - Some functions that were returning `void`, now return `Promise<void>`: `setAllowSharingStoreAccount`, `setFinishTransactions`, `setSimulatesAskToBuyInSandbox`, `addAttributionData`, `setDebugLogsEnabled`, `syncPurchases`, `setAutomaticAppleSearchAdsAttributionCollection`, `invalidatePurchaserInfoCache`, `presentCodeRedemptionSheet`, and all the subscriber attributes related functions.
```
Comment (aboedo) [inline CHANGELOG.latest.md:3]: nitpick: `subscriber attributes-related`
Also, will that be clear to customers at first glance? If they're just using stuff for integrations they might miss that they translate into subscriber attributes, right?

Perhaps a table with all of the affected methods organized by type would make things very easy to understand? 

| Integrations | Subscriber attributes | Configuration and other | 
| :-: | :-: | :-: |
| `addAttributionData` | `collectDeviceIdentifiers` | `invalidatePurchaserInfoCache` |
| `setAd` | `setAttributes` |  `presentCodeRedemptionSheet` |
| `setAdGroup` | `setDisplayName` |  `setAllowSharingStoreAccount` |
| `setAdjustID` | `setEmail` |  `setAutomaticAppleSearchAdsAttributionCollection` |
| `setAirshipChannelID` | `setPhoneNumber` |  `setDebugLogsEnabled` |
| `setAppsflyerID` | `setPushToken` |  `setFinishTransactions` |
| `setCampaign` | |   `setSimulatesAskToBuyInSandbox` |
| `setCreative` | |   `syncPurchases` |
| `setFBAnonymousID` | |  |
| `setKeyword` | | |
| `setMediaSource` | | |
| `setMparticleID` | | |
| `setOnesignalID` | | |

**Review by aboedo — APPROVED** [review]
looks great!! 🎉

File: CHANGELOG.latest.md (hunk 1 above)
Comment (vegaro) [reply CHANGELOG.latest.md:3]: Wow thanks Andy haha. Much better

**Pull request merged**